### PR TITLE
Added hardware_camera package with launch files 

### PR DESCRIPTION
**Pull request by Suryansh470**
I have added hardware_camera with launch files implementing the ROS drive usb_cam for taking the feed ...... also added udev rules and .sh file to copy those udev rules to the required folder of the system...

### CODE REVIEW

File: hardware_layer/hardware_camera/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+* [Logitech C930e](https://www.logitech.com/en-in/product/c930e-webcam)
+
+## Setting up Arduino
+
+### Dependencies
+
+- [Robot Operating System (ROS)](http://wiki.ros.org) (middleware for robotics),
+- Following ROS Packages: [usb_cam](http://wiki.ros.org/usb_cam)
+
+### Preparing the Serial Port
```
Comment (Mayankm96) [inline hardware_layer/hardware_camera/README.md]: Add instructions on setting up `udev` rules here

**Comment by Mayankm96** [comment]
The README has several typo errors. Remove dependencies from the package that are not required.

**Comment by Suryansh470** [comment]
The README.md file is modified .... please review and merge

**Comment by Mayankm96** [comment]
Made minor changes to the PR. Feel free to check them.

**Pull request merged**